eradiate/thermoprops/afgl1986.py

```diff
@@ -92,7 +95,19 @@ def make_profile(
     concentration units, refer to the documentation of
     :func:`~eradiate.thermoprops.util.compute_scaling_factors`.
     """
+    if eradiate.mode().has_flags(ModeFlags.ANY_CKD):
+        if model_id != "us_standard":
+            raise NotImplementedError(
+                "In CKD mode, only the 'us_standard' model is supported."
+            )
+        if concentrations is not None:
+            if any([molecules not in ["H2O", "O3"] for molecules in concentrations]):
```

Comment:
A nicer and efficient way to do so it to use sets. I think something like this should do:
```python
molecules = set(concentrations.keys()) if concentrations else set()

if molecules - {"H2O", "O3"}:
    raise ...
```